### PR TITLE
[IMP] account_bank_statement_clear_partner: Improve README

### PR DESCRIPTION
**Pull request by LudLaf**
Migration to version 15

Readme information enhanced

@Tecnativa TT36411

### CODE REVIEW

File: account_bank_statement_clear_partner/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -1 +1,2 @@
-Enable the **Show Full Accounting Features** option for your user under **Settings** > **Users & Companies** > **Users**
+Enable **Show Full Accounting Features** option for your user under
```
Comment (pedrobaeza) [inline account_bank_statement_clear_partner/readme/CONFIGURE.rst]: Why doing this change that is not changing anything?

**Comment by LudLaf** [comment]
@pedrobaeza please review

File: account_bank_statement_clear_partner/readme/USAGE.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 
-To use this module, you need to:
+To use this module you need to:
```
Comment (pedrobaeza) [inline account_bank_statement_clear_partner/readme/USAGE.rst]: The comma was correct.

**Comment by LudLaf** [comment]
@cesar-tecnativa please review

**Comment by pedrobaeza** [comment]
I still see unneeded changes and the comma is now in an incorrect place.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
We can fast-track this:

/ocabot merge patch

**Pull request merged**